lib/state/util/web3auth-config.ts

```diff
@@ -1,9 +1,32 @@
 import { BaseDotsamaWallet } from "@talismn/connect-wallets";
 import { CHAIN_NAMESPACES } from "@web3auth/base";
-import { Web3Auth } from "@web3auth/modal";
+import { Web3AuthNoModal } from "@web3auth/no-modal";
 import { atom } from "jotai";
+import { isWSX } from "lib/constants";
+import { OpenloginAdapter } from "@web3auth/openlogin-adapter";
+import { CommonPrivateKeyProvider } from "@web3auth/base-provider";
+import { TypeOfLogin } from "@web3auth/openlogin-adapter";
 
-export const clientId = process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID_ZTG;
+interface LoginConfig {
+  [key: string]: {
+    verifier: string;
+    verifierSubIdentifier: string;
+    typeOfLogin: TypeOfLogin;
+    clientId: string;
+  };
+}
+
+export const clientId = isWSX
+  ? process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID_WSX
```

Comment:
Why do we need 2 different variable names? Instead of just `NEXT_PUBLIC_WEB3AUTH_CLIENT_ID`

Reply:
was using it switch between branches for wsx but it's not needed so I removed it